app/components/form/instance.js

```diff
@@ -245,5 +246,8 @@ export default class InstanceComponent extends Component {
     };
     formStore.registerObserver(onFormUpdate);
     onFormUpdate();
+    await timeout(55);
```

Comment:
Seems like a magic number

Reply:
Removed it! I added an await to this.registerObserver and does the same trick